clojurebridge_attendee_care.md

```diff
@@ -0,0 +1,39 @@
+
+1. In our GoogleDrive:
+Leave the original table in its original state.
+Copy it completely and work in the copy.
+
+2. Select attendees by random. To do this generate random numbers from 1 to the amount of applicants. How many get invited depends on how many coaches signed up and is decided by all organizers and clarified before. Add an extra `status` field in the beginning of the table where you note who got `invited`.
+
+3. Send an invite mail to all selected ones. You can use [this template](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#mail-1---to-the-selected) for the mail. You may have to adjust dates and some information. Set a deadline to reply the mail by 3 to 4 days.
+
+4. Send a [mail to the rest telling them they are on the waiting list](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#mail-2---waitinglist).
+
+5. Check the mails frequently. Whenever you hear back from the invited people update their status to either `accepted` or `declined`.
+
+6. If a person declines [invite another random person who wasn't selected](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#inviting-from-waiting-list). Also for them set a deadline for 3 - 4 days.
+
+7. If somebody doesn't get back to any deadline you can send them [a mail](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#to-applicants-who-didnt-get-back) that we gave their spot to the next person.
+
+**Shortly before the event:**
+
+8. When you gathered all the information who attends start to match the groups. It's important to match for language and experience. *We do have attendees who can't speak English. Please consider this carefully.* Also it makes sense to match people who don't want to be photographed.
+
+9. 5 Days before the event: Send another [mail with details about the event](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#mail-3---information-before-the-event).
+
+10. Before the event: prepare name tags with name and photo preference badge.
+
+**At the event:**
+
+11. **Friday:** Do the check in, welcome everybody. See who didn't come. Make sure people feel welcome and have someone to talk too (especially in their language).
+
+12. **Saturday:** Welcome everybody. Check in again. Check who came. Rematch the groups if people don't show up.
+
+13. Wander around the groups. Make sure people feel good. If there are problems, technical or personal take action. Ask floating coaches to jump in or help out. Keep people updated and informed. Remind them to take breaks, keep hydrated and happy.
+
+14. Send a [feedback/wrap-up mail](https://github.com/clojurebridge-berlin/organization/blob/master/email_templates/all_mails_to_attendees.md#mail-4---feedback-and-wrapup) after the event.
```

Comment:
I guess #14 is a new section "after the event"
- send out feedback email
- who accumulates/ compiles the feedback we get? is that also attendee care? 
Also: "Coordinate with the rest of team to make sure attendees are informed of our project groups and Clj meet-ups.